app/src/main/res/xml/preferences.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
     <PreferenceCategory
-        android:title="Location">
+        android:title="Device Registration">
+        <EditTextPreference
+            android:key="registration_name"
+            android:title="Device Name"
+            android:summary="The name of the device listed in Home Assistant"/>
```

Comment:
This summary seems meh, as that's right now a use case, but it could be used for many different things. Better not have any summary ?